torch/csrc/autograd/engine.h

```diff
@@ -179,15 +193,23 @@ struct TORCH_API Engine {
 
 protected:
   void compute_dependencies(Node* root, GraphTask& task);
-  void evaluate_function(NodeTask& task);
+  void evaluate_function(
+      std::shared_ptr<GraphTask> graph_task,
+      Node* func,
+      InputBuffer& inputs);
```

Comment:
Is this diff complete? I expected to see a change to the implementation of `evaluate_function` but I see no other occurrences of `evaluate_function` in this PR

Reply:
Still not fixed. This might be a ghexport problem. I'll go look at the phab diff

Reply:
Hmm, this is really weird. I double checked that the PR was exported correctly. But now magically it shows "no changes" for engine.cpp.